integrations/sd_changed_at.py

```diff
@@ -0,0 +1,165 @@
+import os
+import pickle
+import datetime
+import requests
+import xmltodict
+
+INSTITUTION_IDENTIFIER = os.environ.get('INSTITUTION_IDENTIFIER')
+SD_USER = os.environ.get('SD_USER', None)
+SD_PASSWORD = os.environ.get('SD_PASSWORD', None)
+if not (INSTITUTION_IDENTIFIER and SD_USER and SD_PASSWORD):
+    raise Exception('Credentials missing')
+
+BASE_URL = 'https://service.sd.dk/sdws/'
+
+from_date = datetime.datetime(2019, 2, 15, 0, 0)
+to_date = datetime.datetime(2019, 2, 25, 0, 0)
+
+#from_date = datetime.datetime(2019, 2, 26, 0, 0)
+#to_date = datetime.datetime(2019, 2, 27, 0, 0)
+
+#from_date = datetime.datetime(2019, 2, 27, 0, 0)
+#to_date = datetime.datetime(2019, 2, 29, 0, 0)
+
+#from_date = datetime.datetime(2019, 2, 29, 0, 0)
+#to_date = datetime.datetime(2019, 3, 15, 0, 0)
+
+
+def _sd_lookup(url, from_date=None, to_date=None, params = {}):
+    payload = {
+        'InstitutionIdentifier': INSTITUTION_IDENTIFIER,
+    }
+    payload.update(params)
+
+    full_url = BASE_URL + url
+
+    url_id = url
+    # url_id += 'ActivationDate' + payload['ActivationDate']
+    # url_id += 'DeactivationDate' + payload['DeactivationDate']
+
+    try:
+        with open(url_id + '.p', 'rb') as f:
```

Comment:
Jeg interviewede kort Robert om hvorvidt det var tilrådeligt ikke at kunne konfigurere cachelokationen, og nu ved jeg så at denne er et testprogram, et 'work in progress' - som vi er nødt til at bære med os indtil vi er endeligt i hus med at finde ud af hvad, hvornår og hvor meget vi må slå op i SD.